presto-main/src/main/java/com/facebook/presto/operator/repartition/OptimizedPartitionedOutputOperator.java

```diff
@@ -117,7 +117,8 @@ public OptimizedPartitionedOutputOperator(
                 serdeFactory,
                 sourceTypes,
                 maxMemory,
-                operatorContext.getDriverContext().getLifespan());
+                operatorContext.getDriverContext().getLifespan(),
```

Comment:
Don't pass in the lifeSpan anymore. It's only used in the constructor to create the PartitionBuffers. It can be acquired from operatorContext when creating the PartitionBuffers